test/parallel/test-os.js

```diff
@@ -157,13 +157,13 @@ if (common.isWindows) {
 } else {
   is.number(pwd.uid);
   is.number(pwd.gid);
-  assert.notStrictEqual(pwd.shell.indexOf(path.sep), -1);
+  assert.notStrictEqual(pwd.shell.includes(path.sep), false);
```

Comment:
Good call on the use of `assert.ok`. It's much clearer. I think `!` should be omitted since we're testing  the condition `<cond> !== false`